SmartDeviceLink/SDLLockScreenPresenter.m

```diff
@@ -98,10 +99,11 @@ - (void)sdl_presentIOS13 {
             }
         }
 
+		// Find the currently visible app window
```

Comment:
Indentation

Reply:
Fixed. Apparently in Xcode a tab = 4 spaces and in GitHub 6